elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -393,18 +393,19 @@ default DataStore getDataStore(MetaDataStore metaDataStore, AggregationDataStore
      * @return AggregationDataStore object initialized.
      */
     default AggregationDataStore getAggregationDataStore(QueryEngine queryEngine,
-            Optional<ElideDynamicEntityCompiler> optionalCompiler) {
+            Optional<ElideDynamicEntityCompiler> optionalCompiler, EntityManagerFactory entityManagerFactory) {
         AggregationDataStore aggregationDataStore = null;
-
         if (enableDynamicModelConfig()) {
             Set<Class<?>> annotatedClasses = getDynamicClassesIfAvailable(optionalCompiler, FromTable.class);
             annotatedClasses.addAll(getDynamicClassesIfAvailable(optionalCompiler, FromSubquery.class));
-            aggregationDataStore = new AggregationDataStore(queryEngine, annotatedClasses);
+	    aggregationDataStore = new AggregationDataStore(
+            	() -> { return entityManagerFactory.createEntityManager(); },
+                queryEngine, annotatedClass, (entityManager -> { entityManager.unwrap(Session.class).cancelQuery();}));
         } else {
-            aggregationDataStore = new AggregationDataStore(queryEngine);
+            aggregationDataStore = new AggregationDataStore(
+            	() -> { return entityManagerFactory.createEntityManager(); },
+                queryEngine, (entityManager -> { entityManager.unwrap(Session.class).cancelQuery();}));
```

Comment:
We define the transaction cancel method four times.  Let's define this once with a static, final variable and just use that everywhere.